openapi/components/schemas/GatewayAccountConfig/BVNK.yaml

```
@@ -26,3 +26,13 @@ allOf:
           - merchantId
           - hawkAuthId
           - hawkAuthKey
+      settings:
+        type: object
+        description: BVNK settings object.
+        properties:
+          payoutCurrency:
+            type: string
+            description: Payout cryptocurrency.
```

suggestion: for readability
```suggestion
            description: Cryptocurrency of the payout.
```